Sources/Testing/Running/Configuration.swift

```diff
@@ -258,6 +258,44 @@ public struct Configuration: Sendable {
 
   /// The test case filter to which test cases should be filtered when run.
   public var testCaseFilter: TestCaseFilter = { _, _ in true }
+
+  // MARK: - Expectation checking
+
+  /// Whether or not reflection of values in expressions checked by expectations
```

Comment:
Rather than 3 separate properties, how about a `ValueReflectionOptions` structure?

Reply:
I've taken that suggestion, and made the options property `Optional`, with `nil` indicating that reflection is disabled.

Reply:
Makes sense.